.gitignore

```diff
@@ -46,4 +46,5 @@ npm-debug.log
 .env
 
 #elm
-elm-stuff/
\ No newline at end of file
+elm-stuff/
```

Comment:
What `elm-stuff` is happening in the project? 😄

Reply:
No idea how that got into the project, I recall adding `.vscode` because of my launch settings. Never done Elm in my life 😅